src/app/legacy/containers/Metadata/index.test.jsx

```diff
@@ -68,33 +69,35 @@ const MetadataWithContext = ({
   hasAmpPage,
   /* eslint-enable react/prop-types */
 }) => (
-  <ServiceContextProvider service={service} pageLang={lang}>
-    <RequestContextProvider
-      bbcOrigin={bbcOrigin}
-      id={id}
-      isAmp={platform === 'amp'}
-      pageType={pageType}
-      pathname={pathname}
-      service={service}
-      statusCode={200}
-    >
-      <MetadataContainer
-        title={title}
-        lang={lang}
-        twitterHandle={twitterHandle}
-        description={description}
-        openGraphType={openGraphType}
-        aboutTags={aboutTags}
-        mentionsTags={mentionsTags}
-        image={image}
-        imageAltText={imageAltText}
-        imageHeight={imageHeight}
-        imageWidth={imageWidth}
-        hasAppleItunesAppBanner={hasAppleItunesAppBanner}
-        hasAmpPage={hasAmpPage}
-      />
-    </RequestContextProvider>
-  </ServiceContextProvider>
+  <ThemeProvider service={service}>
```

Comment:
Is this another test where you can use our custom render-with-providers (can't remember the name!) + `toMatchSnapshot` (instead of `shouldMatchSnapshot`)

Reply:
Yeppo! I've just pushed a commit where I have refactored my tests this way. Much simpler... Custom render method all the way ⚡️